src/fp/index.ts

```diff
@@ -0,0 +1,16 @@
+import { DDL_BASE_URL, Context, getInstallId } from "../core";
+import { authedFetch } from "../core/utils";
+import { FingerprintComponents } from "./types";
+
+export async function sendClickRequest(ctx: Context, bannerUid: string, fingerprint: FingerprintComponents): Promise<Response> {
+    const url = `${DDL_BASE_URL}/v1/banners/${bannerUid}/taps`;
+    const webInstallId = await getInstallId();
+
+    return await authedFetch(ctx, url, {
```

Comment:
Do we need to await here if we just return the Promise? (can then drop the async function decl)

Reply:
Ah ok so we can't drop as we await the install ID above, ok

Reply:
👍